packages/fast-components-react-base/src/hypertext/hypertext.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import Foundation, { HandledProps } from "../foundation";
+import {IHypertextProps} from "./hypertext.props";
+import {IHypertextClassNameContract, IManagedClasses} from "@microsoft/fast-components-class-name-contracts";
+
+/* tslint:disable-next-line */
+class Hypertext extends Foundation<IHypertextProps & IManagedClasses<IHypertextClassNameContract>,  React.AllHTMLAttributes<HTMLAnchorElement>, {}> {
+    protected handledProps: HandledProps<IHypertextProps & IManagedClasses<IHypertextClassNameContract>> = {
+        managedClasses: void 0,
+    };
+
+    private generateAttributes() {
```

Comment:
You'll see this approach to applying attributes a lot in the previous repo - the reason we did it this way is we didn't have accurate typescript typings for components.

In this project, we made sure to set things up more accurately - you can just add `href={this.props.href}` in the render method itself and remove this method - that will reduce the code clutter of having an extra method

Reply:
@nicholasrice question - if a disabled href is just an anchor without an href, do we want to check for it like this rather than passing straight into the render function?